lib/exfmt/ast/to_algebra.ex

```diff
@@ -354,12 +354,40 @@ defmodule Exfmt.Ast.ToAlgebra do
   end
 
   #
-  # Zero arity qualified function calls
+  # Zero arity type calls
+  #   e.g. String.t
+  #
+  def to_algebra({{:., _, [aliases, :t]}, _, []}, ctx) do
+    zero_arity_call_to_algebra(".t", aliases, ctx)
+  end
+
+  #
+  # Zero arity qualified function calls to alias
+  #   e.g. Mix.env(), List.flatten/1
+  #
+  def to_algebra({{:., _, [aliases = {:__aliases__, _, _}, name]}, _, []}, ctx) do
+    case arity_labeled_function?(ctx) do
```

Comment:
This traverses the entire stack, I think we only need to check the level above.

We might be able to make it even cleaner by pattern matching on something like this:

```elixir
{:/, _, [{:., _, [{:__aliases__, _, _}, name]}, arity]} when is_integer(arity)
```